diff --git a/Silicon/Qemu/SbsaQemu/Library/SbsaQemuHardwareInfoLib/SbsaQemuHardwareInfoLib.c b/Silicon/Qemu/SbsaQemu/Library/SbsaQemuHardwareInfoLib/SbsaQemuHardwareInfoLib.c
index d37a6709c6..7a6cb0c642 100644
--- a/Silicon/Qemu/SbsaQemu/Library/SbsaQemuHardwareInfoLib/SbsaQemuHardwareInfoLib.c
+++ b/Silicon/Qemu/SbsaQemu/Library/SbsaQemuHardwareInfoLib/SbsaQemuHardwareInfoLib.c
@@ -194,8 +194,12 @@ GetCpuTopology (
   ArmMonitorCall (&SmcArgs);
 
   if (SmcArgs.Arg0 != SMC_SIP_CALL_SUCCESS) {
-    DEBUG ((DEBUG_ERROR, "%a: SIP_SVC_GET_CPU_TOPOLOGY call failed. We have no cpu topology information.\n", __func__));
-    ResetShutdown ();
+    DEBUG ((DEBUG_WARN, "%a: SIP_SVC_GET_CPU_TOPOLOGY unsupported, using fallback topology.\n", __func__));
+    CpuTopo->Sockets  = 1;
+    CpuTopo->Clusters = 1;
+    CpuTopo->Cores    = 4;
+    CpuTopo->Threads  = 1;
+    //ResetShutdown ();
   } else {
     CpuTopo->Sockets  = SmcArgs.Arg1;
     CpuTopo->Clusters = SmcArgs.Arg2;
